chain/stmgr/supply.go

```diff
@@ -303,11 +303,14 @@ func getFilPowerLocked(ctx context.Context, st *state.StateTree) (abi.TokenAmoun
 	return pst.TotalLocked()
 }
 
-func GetFilLocked(ctx context.Context, st *state.StateTree) (abi.TokenAmount, error) {
-
-	filMarketLocked, err := getFilMarketLocked(ctx, st)
-	if err != nil {
-		return big.Zero(), xerrors.Errorf("failed to get filMarketLocked: %w", err)
+func GetFilLocked(ctx context.Context, st *state.StateTree, height abi.ChainEpoch) (abi.TokenAmount, error) {
+	filMarketLocked := big.Zero()
+	if height <= build.UpgradeAussieHeight {
```

Comment:
how about we switch this around with a view to the future, where the common case is `height > build.UpgradeAussieHeight` and comes first, where we just do a simple one-line `return getFilPowerLocked(ctx, st)`, then we don't need `filMarketLocked=0`.